lib/Sema/CSClosure.cpp

```diff
@@ -510,6 +510,56 @@ class ClosureConstraintGenerator
     }
   }
 
+  Optional<SolutionApplicationTarget>
+  getTargetForPattern(PatternBindingDecl *patternBinding, unsigned index,
+                      Type patternType) {
+    auto hasPropertyWrapper = [&](Pattern *pattern) -> bool {
+      auto *singleVar = pattern->getSingleVar();
+      if (!singleVar)
+        return false;
+
+      auto wrapperAttrs = singleVar->getAttachedPropertyWrappers();
+      return !wrapperAttrs.empty();
```

Comment:
You can use `singleVar->hasAttachedPropertyWrapper()` here instead, which does the same thing. It also bails out early if there isn't a custom attribute attached to avoid going into the request evaluator in simple cases.

Reply:
Thanks!

Reply:
Fixed!